src/timer/Timer.js

```diff
@@ -1,17 +1,19 @@
-function Timer(seconds) {
-  this.seconds = seconds;
-}
+class Timer {
+  constructor(seconds) {
+    this.seconds = seconds;
+  }
 
-Timer.prototype.start = function() {
-  var instance = this;
-  var timerInterval = setInterval(function() {
-    if (instance.seconds === 0) {
-      clearInterval(timerInterval);
-    }
+  start() {
+    const timerInterval = setInterval(() => {
+      // this = new (the object we're creating)||calling object (left of .)||bind()||global
```

Comment:
This still refers to out timer class and not the timer interval since we defined it with an arrow function!